OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -33,28 +33,8937 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalQozmos">Salām-hymn to
+                                            Qozmos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.9">
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalYohannes">Salām-hymn to
+                                            Yoḥannǝs</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.10">
+                                        <locus from="97rb" to="97rc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalEsmnunay">Salām-hymn to
+                                            the martyrs of ʾƎsmunayn</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i1.4.22">
+                                    <locus from="97rc" to="97vc"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Heda22"/>
+
+                                    <msItem xml:id="ms_i1.4.22.1">
+                                        <locus from="97rc" to="97vc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Heda22Qozmos"
+                                            >Commemoration of the Tewodyādā and her children</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.22.2">
+                                        <locus target="#97vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda22ComQozmos">Commemoration
+                                            of the companions of Qozmos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.22.3">
+                                        <locus target="#97vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda22Qalementos">Commemoration
+                                            of Qalemǝnṭos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.22.4">
+                                        <locus target="#97vc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda22SalQozmos">Salām-hymn to
```

Comment:
ref is correct? "SalQozmos" not: SalTewodyādā

Reply:
Yes, same as above. And here I added the missing "the", my God...